docs/topics/configmap-mission-resources.adoc

```diff
@@ -1,7 +1,7 @@
 [[about_configmap]]
-= ConfigMap Resources
+= Externalized Configuration Resources
 
-More background and related information on ConfigMap can be found here:
+More background and related information on Externalized Configuration and ConfigMap can be found here:
```

Comment:
Is there a reason why not use `{name-mission-configmap}`? If not, I would advocate for all lower case since it's a common term.

Reply:
"externalized configuration" is a common language, but ConfigMap is Kubernetes-specific term.